administrator/language/en-GB/plg_editors_tinymce.ini

```
@@ -44,6 +44,12 @@ PLG_TINY_FIELD_HTMLWIDTH_LABEL="HTML Width"
 PLG_TINY_FIELD_LABEL_ADVANCEDPARAMS="Advanced"
 PLG_TINY_FIELD_LANGCODE_LABEL="Language Code"
 PLG_TINY_FIELD_LANGSELECT_LABEL="Automatic Language Selection"
+PLG_TINY_FIELD_LINK_CLASS_LIST_DESC="The classes that wil be added to the link. Separate individual classes by a space (example: btn btn-primary). Only valid class names are allowed."
```

spelling wil ==> will
But I think it is better to say "may"
```suggestion
PLG_TINY_FIELD_LINK_CLASS_LIST_DESC="The classes that may be added to the link. Separate individual classes by a space (example: btn btn-primary). Only valid class names are allowed."
```